main/src/mill/modules/Jvm.scala

```diff
@@ -762,6 +769,52 @@ object Jvm {
     }
   }
 
+  @deprecated(
+    "Use alternative overload. This one is only for binary backwards compatibility.",
+    "mill after 0.10.0"
+  )
+  def resolveDependencies(
+       repositories: Seq[Repository],
+       deps: IterableOnce[coursier.Dependency],
+       force: IterableOnce[coursier.Dependency],
+       sources: Boolean,
+       mapDependencies: Option[Dependency => Dependency],
+       customizer: Option[coursier.core.Resolution => coursier.core.Resolution],
+       ctx: Option[mill.api.Ctx.Log]
+   ): Result[Agg[PathRef]] =
+    resolveDependencies(
+      repositories = repositories,
+      deps = deps,
+      force = force,
+      sources = sources,
+      mapDependencies = mapDependencies,
+      customizer = customizer,
+      coursierCacheCustomizer = None,
+      ctx = ctx
```

Comment:
`coursierCacheCustomizer` needs to come after `ctx`, to keep the order of other overrides, which previously had default params.